apps/condo/domains/ticket/schema/Ticket.js

```diff
@@ -677,6 +679,13 @@ const Ticket = new GQLListSchema('Ticket', {
                 },
             },
         },
+        isCompletedAfterDeadline: {
```

Comment:
mb `isExpired`?

Reply:
Should we save true value when ticket returns to work (completed -> inProgress)?

Reply:
now `isCompletedAfterDeadline` tracks `deadline` and `completed` fields